src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService users = new UserService();
+        User found = users.findByEmail(email);
+        if(found == null){
+            return false;
+        }else{
+            return found.getPassword().equals(password);
+        }
```

Comment:
```suggestion
        User foundUser = users.findByEmail(email);
        return foundUser != null && foundUser.getPassword().equals(password);
```
Let's make your code more readable